security/middleware.py

```
@@ -0,0 +1,26 @@
+from .models import AuditLog
+import uuid
+
+class AuditMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
```

Ruff (0.12.2)

Missing return type annotation for special method `__call__`

(ANN204)